renovate.json

```diff
@@ -22,9 +22,13 @@
         "patch",
         "pin",
         "digest"
-      ],
+      ]
```

Comment:
Imagine using json as a configuration language

Reply:
Yeah, I'm bad at it. I _think_  I fixed it (or pre-commit did)